environment-dev.yml

```diff
@@ -0,0 +1,12 @@
+name: murefi-env
+channels:
+- conda-forge
```

Comment:
Do you need conda forge here if you use mamba now?

Reply:
I think channels are mandatory for an `environment.yml`?

And this makes it usable for humans too.